logging.go

```diff
@@ -1,13 +1,30 @@
 package utp
 
 import (
-	"log"
+	"fmt"
 	"os"
+	"time"
+
+	"github.com/anacrolix/log"
 )
 
 const (
 	logCallbacks = false
 	utpLogging   = false
 )
 
-var Logger = log.New(os.Stderr, "go-libutp: ", log.LstdFlags|log.Lshortfile)
+var Logger = log.Logger{log.StreamLogger{
```

Comment:
I think just log.Default.WithPrefixText("go-libutp") will do this.

Reply:
I've updated it to use `log.Default.WithContextText("go-libutp")` (the `WithPrefixText` method doesn't seem to exist).